source/dos.h

```diff
@@ -91,6 +91,7 @@ int installuserfont( char const* filename );
 
 
 enum {
+    DEFAULT_SOUNDBANK_NONE  = 0,
```

Comment:
I think this one, I will move to the implementation section as a #define DEFAULT_SOUNDBANK_NONE 0
The reason being, that the value 0 is intended to be reserved for failure code of installusersoundbank (not that it is documented or anything yet. one day though), and if  you pass 0 to setsoundbank it will do nothing, just ignore the call. Having it exposed in the enum sort of communicates that it is a valid  value to pass to setsoundbank. I can see the use of having it named in the implementation side though, especially with the new delayed loading of soundbanks. But yeah, I think I will move it there

Reply:
Oh, sorry about that, I didn't realize that enum was in the public part of the API.
I was just happy I lowered most of the samples by 1 MB. I changed it to a #define now with 164dae2.